File: docs/legacy-csharp/Gestion de Equipaje y Abordaje/NodoPasajero.cs
public class NodoPasajero
{
    private Pasajero pasajero;
    private NodoPasajero siguiente;

    public Pasajero Pasajero { get => pasajero; set => pasajero = value; }
    public NodoPasajero Siguiente { get => siguiente; set => siguiente = value; }


    public NodoPasajero(Pasajero pasajero)
    {
        Pasajero = pasajero;
        Siguiente = null;
    }
}

File: src/main/java/co/edu/upb/trenes/models/abordaje/NodoPasajero.java
package co.edu.upb.trenes.models.abordaje;

public class NodoPasajero {
    private PasajeroPrioridad pasajero;
    private NodoPasajero siguiente;

    public NodoPasajero(PasajeroPrioridad pasajero) {
        this.pasajero = pasajero;
    }

    public PasajeroPrioridad getPasajero() { return pasajero; }
    public NodoPasajero getSiguiente() { return siguiente; }
    public void setSiguiente(NodoPasajero siguiente) { this.siguiente = siguiente; }
}
